robot-tests/setup_venv.py

```
@@ -0,0 +1,193 @@
+import sys
+import os
+import venv
+
+def run_command(command, description):
+    """Run a command and handle errors"""
+    print(f"🔄 {description}...")
+    try:
+        if isinstance(command, list):
+            result = subprocess.run(command, check=True, capture_output=True, text=True)
+        else:
+            result = subprocess.run(command, shell=True, check=True, capture_output=True, text=True)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)